packages/firebase_dynamic_links/lib/src/firebase_dynamic_links.dart

```diff
@@ -33,7 +33,7 @@ class FirebaseDynamicLinks {
   Future<PendingDynamicLinkData> getInitialLink() async {
     final Map<String, dynamic> linkData =
         await channel.invokeMapMethod<String, dynamic>(
-            'FirebaseDynamicLinks#getInitialLink');
+            'FirebaseDynamicLinks#retrieveDynamicLink');
```

Comment:
I think it'd be easier to just rename the method to `case "FirebaseDynamicLinks#getInitialLink"` in the .java file; that way you shouldn't have to touch this .dart or the iOS plugin, right?